static/bidder-info/ads_interactive.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://bntb.adsintreactive.com/"
+maintainer:
+  email: "it@adsinteractive.com"
```

Comment:
I have sent a email, please confirm

Reply:
Please write back to my email. We need to confirm that this email works
![image](https://github.com/user-attachments/assets/f9d4f645-0702-407c-a515-9e6bb2fb9f5b)

Reply:
confirmed:
![image](https://github.com/user-attachments/assets/1fdfe7a0-d2d9-42d2-bf15-88df6306e21d)